### PR TITLE
adding powershell and fish and modifying helpers

### PR DESCRIPTION
**Pull request by viniciusramosdefaria**
**- What I did**
Added missing helpers and autocomplete

**- How to verify it**
See if `rit completion` reflects:
```
Add autocomplete for terminal, Available for (zsh, bash, fish, powershell).

Examples:
rit completion zsh

Available Commands:
  bash        Add bash autocomplete for terminal, --help to know how to use
  fish        Add fish autocomplete for terminal, --help to know how to use
  powershell  Add powerShell autocomplete for terminal, --help to know how to use
  zsh         Add zsh autocomplete for terminal, --help to know how to use

```

### CODE REVIEW

File: pkg/cmd/autocomplete.go [hunk 1]
```diff
@@ -40,9 +40,19 @@ func NewAutocompleteZsh(g autocomplete.Generator) *cobra.Command {
...
+		Short:   "Add zsh autocomplete for terminal, --help to know how to use",
+		Long:    `
+Add bash autocomplete for terminal
+Only works if zsh auto completion is installed.
+
+To test run: 
+ $ rit completion zsh | source
+
+To install run: 
+ $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion zsh > ~/.rit_completion" >> ~/.zshrc
```
Comment (viniciussousazup) [inline pkg/cmd/autocomplete.go:52]:
```suggestion
 $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion zsh | source" >> ~/.zshrc
```

i guess only save completion on ~/.rit_completion don't work if the user not call "source ~/.rit_completion  "

File: pkg/cmd/autocomplete.go [hunk 2]
```diff
@@ -53,9 +63,19 @@ func NewAutocompleteBash(g autocomplete.Generator) *cobra.Command {
...
+		Short:   "Add bash autocomplete for terminal, --help to know how to use",
+		Long:    `
+Add bash autocomplete for terminal
+Only works if bash auto completion is installed.
+
+To test run: 
+ $ rit completion bash | source
+
+To install run: 
+ $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion bash > ~/.rit_completion" >> ~/.bashrc
```
Comment (viniciussousazup) [inline pkg/cmd/autocomplete.go:75]:
```suggestion
 $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion bash | source" >> ~/.bashrc
```
i guess only save completion on ~/.rit_completion don't work if the user not call "source ~/.rit_completion  "

File: pkg/cmd/autocomplete.go [hunk 3]
```diff
@@ -40,9 +40,19 @@ func NewAutocompleteZsh(g autocomplete.Generator) *cobra.Command {
 
 	return &cobra.Command{
 		Use:     zsh.String(),
-		Short:   "Add zsh autocomplete for terminal",
-		Long:    "Add zsh autocomplete for terminal",
-		Example: "rit completion zsh",
+		Short:   "Add zsh autocomplete for terminal, --help to know how to use",
+		Long:    `
+Add bash autocomplete for terminal
```
Comment (viniciussousazup) [inline pkg/cmd/autocomplete.go:45]:
```suggestion
Add zsh autocomplete for terminal
```

File: pkg/cmd/autocomplete.go (hunk 2 above)
Comment (viniciusramosdefaria) [reply pkg/cmd/autocomplete.go:76]: I added the line:

` echo "source ~/.rit_completion" >> ~/.bashrc`
To solve the missing stuff.
The reason we need to do this way is that when running on macOS stuff breaks if you do like you mentioned. I don't know why LOL.

File: pkg/cmd/autocomplete.go (hunk 1 above)
Comment (viniciusramosdefaria) [reply pkg/cmd/autocomplete.go:52]: I added the line:

` echo "source ~/.rit_completion" >> ~/.zsh `

To solve the missing stuff.
The reason we need to do this way is that when running on macOS stuff breaks if you do like you mentioned. I don't know why LOL.

**Pull request merged**